clients/cli/README.md

```diff
@@ -71,6 +70,46 @@ To run the CLI with tracing enabled, run:
 cargo run -- beta.orchestrator.nexus.xyz
 ```
 
+### Building a release binary
+
+```sh
+# Build a release binary for Apple Silicon (M1/M2) Mac
+cargo build --release --target aarch64-apple-darwin
+
+# Build a release binary for Intel Mac
+cargo build --release --target x86_64-apple-darwin
+
+# Create a tarball for Apple Silicon (M1/M2) Mac binary and return to original directory
+cd target/aarch64-apple-darwin/release && tar -czf aarch64-apple-darwin.tar.gz ./prover && cd -
+
+# Create a tarball for Intel Mac binary and return to original directory
+cd target/x86_64-apple-darwin/release && tar -czf x86_64-apple-darwin.tar.gz ./prover && cd -
+
+# Test the binary (use appropriate path based on target)
+./target/aarch64-apple-darwin/release/prover --version 
+
+# Run the binary (use appropriate path based on target)
+./target/aarch64-apple-darwin/release/prover beta.orchestrator.nexus.xyz
+```
+
+Note: Make sure you have the appropriate target installed via rustup:
+```sh
+# Add required target(s)
+rustup target add aarch64-apple-darwin
+rustup target add x86_64-apple-darwin
+rustup target add x86_64-unknown-linux-gnu
+rustup target add aarch64-unknown-linux-gnu
+```
+
+### Clearing Nexus data
+
+To completely reset Nexus for a clean installation, run these commands:
+
+```sh
+# Remove the entire Nexus directory
+rm -rf ~/.nexus
```

Comment:
Can we not remove the entire .nexus directory? There may be some valuable backups there.